include/roboteam_ai/bt/Node.hpp

```diff
@@ -87,6 +87,8 @@ class Node {
 
 //        ros::Time getLastTickTime();
 
+        void setRole(std::string roleName);
```

Comment:
Maybe name this function setRoleString() or so, so that the difference between the class Node and the function is clear for outside users.

Reply:
Well yes but we usually just have a literal in node_name